src/formatter.ts

```diff
@@ -305,15 +305,15 @@ function* formatNode(n: Node, o: Options = {}) {
       break;
     }
     case 'strong': {
-      yield '**';
+      yield n.attributes.marker ?? '**';
       yield* formatInline(formatChildren(n, no));
-      yield '**';
+      yield n.attributes.marker ?? '**';
       break;
     }
     case 'em': {
-      yield '_';
+      yield n.attributes.marker ?? '*';
```

Comment:
Why are we changing `_` to `*`?

Reply:
Is there any harm in leaving it as `_` (as a default _convention_)?

Reply:
Yeah, so the issue is that `_` is ignored when it's used inside of a text node, e.g. `foo_bar_baz` is  rendered literally whereas `foo*bar*baz` treats the `*bar*` part as emphasized. Because the underscore behaves differently in this specific case, it's not safe to convert `foo*bar*baz` to `foo_bar_baz`—it changes the meaning of the content. So using the asterisk is a safer universal default than the underscore. It's a little unfortunate, I'd prefer the underscore otherwise.

Reply:
Ahhh, makes sense.